reference/spender.yaml

```diff
@@ -3234,6 +3234,12 @@ components:
           description: |
             If the transaction is created by statement uploaded then this key refers to the source statement.
           example: stmtdasfdsaf3
+        can_delete:
```

Comment:
@bewithgaurav sir, added one minor comment here, also is it possible to have this api only for admin and ignore for spender?